test/heartbeat_test.py

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env/python3
+
+import sys
+import subprocess
+
+argv = sys.argv
+node0 = subprocess.Popen([argv[1], "0"])
+node1 = subprocess.Popen([argv[1], "1"])
+node0.kill()
```

Comment:
The defaut multi_process_test will exit very quickly, you will likely miss chance to kill it.

You can write another case in which the job will run forever.